<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Address :</string>
   </property>
   <property name="buddy">
    <cstring>label</cstring>
   </property>
  </widget>
  <widget class="QPushButton" name="puB_Listen">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>70</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Port :</string>
   </property>
   <property name="buddy">
    <cstring>label</cstring>
   </property>
  </widget>
  <widget class="QTextEdit" name="receive_text">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>170</y>
     <width>331</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="sent_text">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>360</y>
     <width>331</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="puB_Sent">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>480</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="AddressEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>127.0.0.1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PortEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>1234</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Receive :</string>
   </property>
   <property name="buddy">
    <cstring>label</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sentï¼š</string>
   </property>
   <property name="buddy">
    <cstring>label</cstring>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>puB_Listen</zorder>
  <zorder>label_2</zorder>
  <zorder>receive_text</zorder>
  <zorder>AddressEdit</zorder>
  <zorder>PortEdit</zorder>
  <zorder>sent_text</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>puB_Sent</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
